resources/windows/wix/service.puppet.wxs.erb

```diff
@@ -2,18 +2,12 @@
 <Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
 
   <Fragment>
-    <ComponentGroup Id="Service_PUPPET">
-      <ComponentRef Id="PuppetServiceAutomatic" />
-      <ComponentRef Id="PuppetServiceManual" />
-      <ComponentRef Id="PuppetServiceDisabled" />
-    </ComponentGroup>
 
-    <DirectoryRef Id="PUPPETBINDIR">
-      <!--Seriously?... http://stackoverflow.com/questions/9419411/wix-setting-service-startup-type-using-a-property-property-not-recognized -->
-      <!-- Yep, Seriously - http://sourceforge.net/mailarchive/forum.php?thread_name=CANJN1a6gDE2tUUnP4SE05m8Ojh2q0Y8LrWOxbrF89YyJJCHn1A%40mail.gmail.com&forum_name=wix-users -->
+    <ComponentGroup Id="<%= get_service("puppet").component_group %>">
       <Component Id='PuppetServiceAutomatic'
-                 Guid="639ECD7F-6186-43D5-9E1A-FC0278DBEE15"
-                 Win64="<%= settings[:win64] %>">
+                  Guid="639ECD7F-6186-43D5-9E1A-FC0278DBEE15"
+                  Directory="<%= get_service("puppet").bindir_id %>"
+                  Win64="<%= settings[:win64] %>">
```

Comment:
I don't know if you care, but there seems to be an extra space here (lines 8-10) before the `Guid=`, etc, where as all the other pieces in this format have their options all lined up